README.md

```diff
@@ -1,2 +1,31 @@
 # Hello-Microverse
 Linters test for HTML and CSS
+
+Project Name
+Linters Test
+
+Built With
+HTML and CSS
+Technologies used
+
+
+To get a local copy up and running follow these simple example step.
+1) Clone into your local machine
+
+Prerequisites
+Setup Git
+Install code editor(Vs code; highly recommended)
+
+👤 Author1
+
+GitHub: Ade179
+Twitter: @Juwon_Adesanya
+LinkedIn: Juwon Adesanya
+
+
+
+Show your support
+Give a ⭐️ if you like this project!
+
+📝 License
+This was issued by microverse
```

Comment:
- [ ] Update your readme file to make it more professiona, use this [Template](https://github.com/microverseinc/readme-template)